README.md

```diff
@@ -1,119 +1,105 @@
+Our technical interview involves pairing with two of our engineers and working through a ticket we've completed recently. Our goal is to give you enough time so you are able to ask questions, get the necessary context needed, and make progress on the problem, ideally without feeling like you are racing against the clock.
 
-# Candidate Technical Interview Prep Guide:
+# Technical Interview Prep Guide
 
-At Home Chef our tech stack is a mix of React, Redux HTML, CSS (tailwinds), HAML/Rails Views, and JavaScript on the front end. We use a Ruby on Rails back end with a PostgreSQL database. We don't expect you to be an expert in these things.
+At Home Chef our tech stack is a mix of React, Redux, HTML, CSS (Tailwind), HAML/Rails views, and JavaScript on the front end. We use a Ruby on Rails back end with a PostgreSQL database. We don't expect you to be an expert in these things.
 
 Our technical interview is designed so you can make progress on it regardless of your specific technical background or level of experience. That being said, being familiar with the concepts and tools below may be helpful during the technical interview:
-  - Basic `Ruby on Rails` CRUD app functionality
-  - Interacting with `API's` (Making GET/POST requests)
-  - `React`
-  - `React Hooks`, specifically `useState`
-  - `Redux/State Management`
-  - `Serializers`
-
-# Candidate Technical Interview Setup Guide:
-# Mac Setup:
-
-**TODO BEFORE THE INTERVIEW:**
-- Install [VS code](https://code.visualstudio.com/Download)
-- Add/install Microsoft's "[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)" extension
-
-**TODO WITH YOUR INTERVIEWER:**
-- Create ssh config file *if one does not already exist*
-	- Navigate to root directory '~'. `cd ~`
-    If `.ssh` **DOES NOT** exist:
-    - In root directory (~), `mkdir .ssh` 
-    - `ls .ssh`
-    - Inside `~/.ssh`, `touch config`
-	If `.ssh` **DOES** already exist, proceed to next steps:
-- Open config file
-- Paste connection info into the `config` file and save config file. Your interviewer will send you the connection info, but see example below:
+  - Basic CRUD app functionality
+  - Interacting with APIs (making GET/POST requests)
+  - React and React Hooks, including state management
+
+# Interview Setup Guide
+### Mac Setup
+
+#### To complete before the interview:
+1. Install [VS code](https://code.visualstudio.com/Download).
+2. Add/install Microsoft's "[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)" extension.
+
+#### To complete along with your interviewer:
+1. Create ssh config file *if one does not already exist* (if `.ssh` **DOES** already exist, proceed to Step 2).
```

Comment:
I numbered these so it's a little easier to refer to each step.

Reply:
How about so there is less confusion the inner list be changed from `1.` to `1)` or `(1)`

Reply:
Because of the indent, it shows up as `i.`

Reply:
mmmm. okay. nice!